capstone/src/main/resources/static/event.js

```diff
@@ -17,6 +17,8 @@ function createEvent(event) {
   createEventAttribute(event.eventDescription, listEventElement);
   createEventAttribute(event.eventLatitude, listEventElement);
   createEventAttribute(event.eventLongitude, listEventElement);
+  createEventAttribute(event.foodAvaliable, listEventElement);
+  createEventAttribute(event.requiredFee, listEventElement);
```

Comment:
Very concise and useful helper function!

Reply:
^

Reply:
You might consider having a common library of useful utilities, shared across different pages.